plugins/ui/src/js/src/widget/WidgetErrorView.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '@deephaven/components';
+import ErrorView from './ErrorView';
+import { WidgetError } from './WidgetTypes';
+
+/** Component that takes a WidgetError and displays the contents in an ErrorView, and has a button to reload the widget from a fresh state. */
+function WidgetErrorView({
+  error,
+  onReload: onReset,
+}: {
+  error: WidgetError;
+  onReload: () => void;
+}): JSX.Element {
+  const displayMessage = `${error.message}\n\n${error.stack ?? ''}`.trim();
```

Comment:
Might as well trim the message in case it has trailing whitespace/newlines

```suggestion
  const displayMessage = `${error.message.trim()}\n\n${error.stack ?? ''}`.trim();
```